lib/proc/proc_tokenize.c

```diff
@@ -54,6 +54,29 @@ fin_tokens(grn_ctx *ctx, grn_obj *tokens)
   GRN_OBJ_FIN(ctx, tokens);
 }
 
+static void
+output_tokens_simple(grn_ctx *ctx, grn_obj *tokens, grn_obj *lexicon)
+{
+  size_t i;
+  size_t n_tokens = GRN_BULK_VSIZE(tokens) / sizeof(tokenize_token);
+
+  grn_ctx_output_array_open(ctx, "TOKEN_VALUES", (int)n_tokens);
```

Comment:
Why do you use `TOKEN_VALUES` not `TOKENS`?

Reply:
I chose `TOKEN_VALUES` instead of `TOKENS` to make it clear that we’re outputting only the value field of each token. A full token object includes properties like `position` and `force_prefix`, but TOKEN_VALUES emphasizes that only the raw strings are shown.

```
tokenize   --tokenizer TokenMecab   --string 'Groongaはオープンソースのカラムストア機能付き全文検索エンジンです。'   --output_style full
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "Groonga",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    },
    ...
  ]
]
```

Reply:
I'm rethinking of it. We don't use the `TOKEN_VALUES` word in Groonga code bases. So I think we should use the `TOKEN` and it's easy for us to understand it because `values` is `TOKEN` too.

Reply:
Do we need to emphasize it here? Which case is this helpful?
FYI: Array name (`TOKEN_VALUES`) is used for XML output.

I feel that it's better that we choose shorter array name (`TOKENS`) rather than longer array name (`TOKEN_VALUES`) for "simple" style.

Reply:
> I feel that it's better that we choose shorter array name (TOKENS) rather than longer array name (TOKEN_VALUES) for "simple" style.

Thank you so much, I agreed with your opinion.